src/module/vue/components/xp/legacy-track.vue

```diff
@@ -0,0 +1,140 @@
+<template>
```

Comment:
defer to another PR

Reply:
on second thought, maybe not. it could use the old xp widget for now and just update the progress element